Documentation/Patterns.md

```diff
@@ -33,8 +33,8 @@ ErrorOr<NonnullRefPtr<Bitmap>> Bitmap::create_shareable(BitmapFormat format, Int
     auto const pitch = minimum_pitch(size.width() * scale_factor, format);
     auto const data_size = size_in_bytes(pitch, size.height() * scale_factor);
 
-    auto buffer = TRY(Core::AnonymousBuffer::create_with_size(round_up_to_power_of_two(data_size, PAGE_SIZE)));
-    auto bitmap = TRY(Bitmap::create_with_anonymous_buffer(format, buffer, size, scale_factor, {}));
```

Comment:
I'm not sure I understand the motivation behind renaming Anonymous buffer. One of the main features of anonymous buffer is that it models the "memfd" concept in linux, where the only actors that have access to the data are ones with whom a handle has been shared specifically.

Reply:
When I first saw this name, I was confused: aren't all buffers anonymous? Surely all buffers I encountered before were anonymous.
Yes, one of the features is that it models memfd. But the main feature, its primary purpose is shared buffer. memfd is just an implementation detail.
And by the way, even not considering Windows implementaion, anon_create on some systems (BSD generic, Haiku) is implemented using named shm_open.

To summarize: the name AnonymousBuffer is confusing, doesn't reflect its primary purpose and just plain wrong on some platforms.

Reply:
No response => failed to convince => no renaming